config/initializers/devise.rb

```diff
@@ -23,6 +23,8 @@
 
   config.sign_out_via = :delete
 
+  config.timeout_in = 30.minutes
```

Comment:
I think we can probably bump this to let's say `7.days`? Monzo (banking app) definitely keep their session alive for quite a while, though their app does require your PIN / Fingerprint / FaceID if you need to do certain sensitive operations.

30 minutes is probably too short. @fofr is there any guidance you're aware of for expiring sessions and how long they ought to last?

Reply:
Sounds good. We weren't sure how long to keep sessions, so feedback on this is super welcome :)

Reply:
I've spoken to Paul H and 7.days seems adequate for now.